Index.html

```diff
@@ -13,206 +12,222 @@
-      <div class="list_programs">
-        <ul>
-          <li<a><img class="css"src="./images/css.svg"></a></li>
-            <li<a><img class="htm"src="./images/html.svg"></a></li> 
-              <li<a><img class="bot"src="./images/boot.svg"></a></li>
-                <li<a><img class="ruby"src="./images/ruby.svg"></a></li>
-        </ul>
-      </div>
-      <a target="_blank" class="b1" href="https://www.microverse.org/">See Project</a> 
-    
-      <div class="grid">
-      <div class="projects">
-        <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
-          <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-        
-          <div class="list_programs2">
-          <ul>
-            <li<a><img class="htm"src="./images/html2.svg"></a></li> 
-              <li<a><img class="bot"src="./images/boot2.svg"></a></li>
-                <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+      <nav class="headerP2">
+        <h2 class="tituloMR">My Recent Works</h2>
+        <img class="vector" src="./images/Vector.svg">
+      </nav>
+
+      <div class="bodyP2">
+        <img class="yoga" src="./images/Img Placeholder.svg">
+
+      <div class="textright">
+          <h3 class="multi">Multi-Post Stories</h3>
+          <p class="parrafo1">A daily selection of privately personalized reads; no accounts or sign-ups required. has
+            been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy
+            text.</p>
+          <ul class="list_programs">
+            <li<a><img class="css" src="./images/css.svg"></a></li>
+              <li<a><img class="htm" src="./images/html.svg"></a></li>
+                <li<a><img class="bot" src="./images/boot.svg"></a></li>
+                  <li<a><img class="ruby" src="./images/ruby.svg"></a></li>
           </ul>
+          <a target="_blank" class="b1">See Project</a>
         </div>
-        <a target="_blank" class="b2" href="https://www.microverse.org/">See Project</a>
-        </div>
 
-        <div class="projects">
-          <h2 class="tituloPAPD">Profesional Art Printing Data</h2>
-            <p class="parrafo2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-          <div class="list_programs2">
-            <ul>
-              <li<a><img class="htm"src="./images/html2.svg"></a></li> 
-                <li<a><img class="bot"src="./images/boot2.svg"></a></li>
-                  <li<a><img class="ruby"src="./images/ruby2.svg"></a></li>
+      </div>
+
+
+        <div class="gridprojects">
```

Comment:
- [ ] [OPTIONAL]Kindly change this to a `section` tag. It is the correct semantic and screen readers know where it starts. a `div` is too general